uwg_schema/model.py

```diff
@@ -236,25 +223,43 @@ class UWG(NoExtraBaseModel):
         'pedestrians, and street cooking.'
     )
 
-    bld: conlist(conlist(float, min_items=3, max_items=3), min_items=1) = Field(
+    bld: conlist(
+        conlist(
+            Union[float, str], min_items=3, max_items=3),
+        min_items=1) = Field(
         ...,
-        description='Matrix of numbers representing fraction of urban building stock. '
-        'This property consists of a 16 x 3 matrix referencing the fraction of the '
-        'urban building stock from 16 building types and 3 built eras representing, in '
-        'combination with 16 climate zones, 768 building archetypes generated from the '
-        'Commercial Building Energy Consumption Survey. Each column represent a '
-        'pre-1980s, post-1980s, or new construction era, and rows represent building '
-        'types. Custom build types can be added by adding new rows. The sum of the '
-        'fractional values in the bld matrix must sum to one.'
+        description='List of building types, eras, and fractions of urban building '
+        'stock. This consists of a list of tuples, each containing a string for the '
```

Comment:
The distinction between lists and tuples doesn't exist in JSON.  It's probably best to just use the generic term "array".